airmozilla/popcorn/views.py

```diff
@@ -0,0 +1,48 @@
+import requests
+
+from django.shortcuts import get_object_or_404
+
+from jsonview.decorators import json_view
```

Comment:
`jsonview` and `requests` are non-django third-party libs so they belong together grouped as import statements.